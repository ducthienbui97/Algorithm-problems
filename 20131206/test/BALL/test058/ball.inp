7 848376 905498 95561 301837
342815 232180 755111 38747 594928 618163 283811 403378 968815 857911 921275 588145 944986 722882 434124 319788 295777 445439 570706 64801 799305 550984 338196 871271 211781 351928 863841 907362 984689 981084 213609 366692 689350 395533 602885 170258 275372 392922 786243 572126 609306 546534 806761 33597 437961 452028 65709 23242 22896 
666169 546990 805163 325793 128933 974127 306341 937731 597159 263399 905778 695023 708235 835741 565678 320771 176595 786664 520008 428812 777353 882245 38201 66803 293550 444919 407999 42935 287815 227239 169040 343050 882714 752907 47011 916017 461204 221257 316482 540029 611847 108467 987501 450446 291667 554679 756290 972074 259597 
363872 550661 664503 606250 369745 90728 125711 203982 239874 776392 14334 934747 243250 330858 901112 694328 867387 354494 472012 662158 165969 195524 72882 829370 444613 501901 276053 307171 47442 616504 388535 68349 895279 172624 737764 844982 546538 665000 173997 418424 576466 460001 911302 946681 317677 585078 101264 795965 965372 
646706 408465 779323 237205 6205 728497 933819 29531 629257 565136 732041 687811 996336 572138 62414 466213 238331 421460 610708 185446 593804 734474 549619 963484 324252 418192 185070 791585 868518 122411 722250 624287 442283 85511 837334 214853 11197 426702 429663 490118 540460 507384 648131 599804 906947 4624 58363 440519 482810 567291 528848 890417 931741 756325 45854 913386 35790 158151 814934 323346 174416 799742 359430 938673 104056 917179 735638 974750 804774 823025 230453 752551 894710 621852 490794 183840 581033 4274 298475 290296 244466 895194 348978 14195 351001 790884 342707 210676 922549 15586 981999 457182 716031 612876 690612 616962 188894 784897 502605 384017 453105 703923 238898 389517 703414 453642 472258 486475 895208 402173 373687 617440 201532 345984 924646 672012 238719 98605 978126 954387 13248 318617 431460 785389 961946 535294 871822 745315 23446 17179 190347 757507 847757 361550 261430 886354 976955 940774 449472 171809 256565 125359 352957 520818 170325 762008 579396 262030 168994 103751 433640 701267 774489 278147 621841 806678 704110 489916 575944 251989 558830 550158 239206 886065 870668 91027 33495 272294 721842 430249 592011 482853 934130 935528 706456 219827 227471 660307 484518 638655 704276 65512 559272 686764 74078 834615 508112 677998 230281 589124 896135 201582 385734 997419 549649 141421 927401 23039 15569 533502 229999 293960 51932 655181 777953 14681 406747 242119 285442 685764 620665 127667 298268 540490 754308 901946 44358 352919 388022 688195 573076 852559 416638 511609 814363 237481 411213 374273 228582 771962 393992 831866 878919 450507 401265 100056 407997 493616 26797 948416 442985 57931 871695 943945 435051 674364 504369 35728 450842 853491 64477 976873 412382 259636 15583 487844 408934 658207 521695 704454 975105 848197 758513 558391 101637 27134 919086 955724 720517 750991 317051 789447 341255 55622 741614 520931 438956 226322 931006 515724 196534 338560 99002 372138 247056 943175 218591 513257 66884 797600 764189 835173 453081 366403 325513 872452 491864 600801 375243 717474 274352 609441 233787 678560 670156 27363 670262 287493 271008 229462 300891 561549 92570 950877 845277 193430 32176 719457 574510 987162 774735 950101 134579 559571 506169 794651 333966 636682 109281 185493 436856 495226 371831 280818 244514 313371 288592 836835 762333 486085 83933 916385 600911 