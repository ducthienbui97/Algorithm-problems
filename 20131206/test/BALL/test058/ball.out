11745500291435
